editor/editor_export.cpp

```diff
@@ -1298,6 +1298,31 @@ String EditorExportPlatform::test_etc2() const {
 	return String();
 }
 
+String EditorExportPlatform::test_etc2_or_pvrtc() const {
+
+	String driver = ProjectSettings::get_singleton()->get("rendering/quality/driver/driver_name");
+	bool driver_fallback = ProjectSettings::get_singleton()->get("rendering/quality/driver/fallback_to_gles2");
+	bool etc_supported = ProjectSettings::get_singleton()->get("rendering/vram_compression/import_etc");
```

Comment:
I guess `etc_supported` is not relevant for iOS (and now disabled by `platform/iphone/export/export.cpp` changes), so maybe it can be removed from the checks / error messages?

Reply:
Yes, I think so

Reply:
I pushed an amend that removes the `etc_supported` check.